packages/story-editor/src/components/mediaRecording/footer.js

```diff
@@ -276,6 +283,40 @@ function Footer({ captureImage, videoRef }) {
     speak,
   ]);
 
+  const {
+    actions: { uploadFile },
+  } = useUploader();
+
+  const { updateStory } = useStory(({ actions: { updateStory } }) => ({
+    updateStory,
+  }));
+  const { setHighlights } = useHighlights(({ setHighlights }) => ({
+    setHighlights,
+  }));
+
+  const { convertToMp3 } = useFFmpeg();
+
+  const onAudioInsert = async () => {
+    setIsInserting(true);
+    speak(__('Inserting…', 'web-stories'));
```

Comment:
Are we inserting here? Are we not setting background audio.

Reply:
Inserting as background audio.

The current wording works for me. We can do a holistic review of all media recording labels separately if needed.

Reply:
We are converting the file, uploading (inserting) the file onto the server and setting the audio bg property 🤔 
Setting background audio is just a fraction of the time this label is visible, we are mainly uploading/converting, I guess the upload part is the longest, so maybe `Uploading...` label would be the best? the user would know that it's network-dependent action and can take time. UX should handle this? holistic review +1.